client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/Configuration.java

```diff
@@ -395,6 +399,15 @@ public RemoteCacheManagerMetricsRegistry metricRegistry() {
       return metricRegistry;
    }
 
+   /**
+    * Controls how long a server is marked as failed when using BASIC intelligence in milliseconds.
```

Comment:
I guess we can remove `when using BASIC intelligence` since it is enabled independently of the intelligence.

Reply:
Yeah, I missed updating this one, I updated the other module.